Modelica/Mechanics/Translational/Components/IdealGear.mo

```diff
@@ -0,0 +1,69 @@
+within Modelica.Mechanics.Translational.Components;
+model IdealGear "Ideal gear without mass"
+  extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
+  parameter Real ratio(start=1)
+    "Transmission ratio (flange_a.s/flange_b.s)";
+
+equation
+  if abs(ratio) < Modelica.Constants.eps then
+    flange_a.f = 0;
+    flange_b.f = 0;
+  else
+    s_a = ratio*s_b;
+    0 = ratio*flange_a.f + flange_b.f;
+  end if;
+
+  annotation (
+    Documentation(info="<html>
+<p>
+This element characterizes any type of gear which is fixed in the
+ground and which has one driving flange and one driven flange.
```

Comment:
Almost all elements where it is relevant to have a connection to ground/support/housing has an optional such connector. any particular reason this one is different?

Reply:
No particular reason for this. I simply took and modified documentation of `Modelica.Mechanics.Rotational.Components.IdealGear` which refers to ground in the same manner.